jolt-core/src/jolt/vm/instruction_lookups.rs

```diff
@@ -1003,16 +1106,16 @@ where
             })
             .collect();
 
-        let mut instruction_flag_bitvectors: Vec<Vec<usize>> =
-            vec![vec![0usize; m]; Self::NUM_INSTRUCTIONS];
-        for (j, op) in self.ops.iter().enumerate() {
+        let mut instruction_flag_bitvectors: Vec<Vec<u64>> =
+            vec![vec![0u64; m]; Self::NUM_INSTRUCTIONS];
+        for (j, op) in ops.iter().enumerate() {
             let opcode_index = op.to_opcode() as usize;
             instruction_flag_bitvectors[opcode_index][j] = 1;
         }
 
         let instruction_flag_polys: Vec<DensePolynomial<F>> = instruction_flag_bitvectors
             .par_iter()
-            .map(|flag_bitvector| DensePolynomial::from_usize(&flag_bitvector))
+            .map(|flag_bitvector| DensePolynomial::from_u64(flag_bitvector))
```

Comment:
Does `from_u64` end up using the lookup table?

Reply:
yes, it uses the lookup table for everything from 0 up to 2^14